src/components/Dialogs/SceneDialogs/EditSceneDialog.tsx

```diff
@@ -254,7 +256,9 @@ const EditSceneDialog = () => {
             e.target.value &&
             activatePreset(
               dev,
-              'default_presets',
+              // TODO: Change this from default_presets to either ledfx_presets or user_presets based on the preset that was selected
+              'user_presets',
```

Comment:
This is why the user presets are not working in the scene selector, it's hardcoded currently.

I appear to have inadvertently flipped it to user_presets, but you get the idea.